api/ConstantSourceNode.json

```diff
@@ -112,66 +110,10 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": "52",
-              "version_removed": "53",
-              "notes": "This property is still available, but via the inheritance of <code>AudioScheduledSourceNode</code>."
-            },
-            "firefox_android": {
-              "version_added": "52",
-              "version_removed": "53",
-              "notes": "This property is still available, but via the inheritance of <code>AudioScheduledSourceNode</code>."
-            },
-            "ie": {
-              "version_added": false
-            },
-            "opera": {
-              "version_added": false
-            },
-            "opera_android": {
-              "version_added": false
-            },
-            "safari": {
-              "version_added": false
-            },
-            "safari_ios": {
-              "version_added": false
-            },
-            "samsunginternet_android": {
-              "version_added": false
-            },
-            "webview_android": {
-              "version_added": false
-            }
-          },
-          "status": {
-            "experimental": false,
-            "standard_track": true,
-            "deprecated": false
-          }
-        }
-      },
-      "onended": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ConstantSourceNode/onended",
-          "support": {
-            "chrome": {
-              "version_added": "56"
-            },
-            "chrome_android": {
-              "version_added": "56"
-            },
-            "edge": {
-              "version_added": "79"
-            },
-            "firefox": {
-              "version_added": "52",
-              "version_removed": "53",
-              "notes": "This property is still available, but via the inheritance of <code>AudioScheduledSourceNode</code>."
+              "version_added": "52"
```

Comment:
The diff here looks weird, but it is just a removal of some entries, I didn't change `offset`. (The data for it is wrong though. Not gonna fix it here.)